pkg/provider/aci.go

```diff
@@ -484,7 +484,15 @@ func (p *ACIProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 		return err
 
 	}
+
+	// get initContainers
+	initContainers, err := p.getInitContainers(pod)
```

Comment:
Please pass the ctx on each new func and use it to log error

Reply:
logging error messges